### PR TITLE
Added serverBind, mysqlHost, mysqlPort params

### PR DESCRIPTION
**Pull request by asiragusa**

### CODE REVIEW

**Comment by menicosia** [comment]
Hi @asiragusa, just FYI we really can't begin to engage on this PR till you've signed the CLA, per above.

**Comment by asiragusa** [comment]
Hi @menicosia why aren't you releasing this code on a standard OSS licence?

**Comment by robdimsdale** [comment]
@asiragusa The license should be Apache 2; the lack of a LICENSE file is an oversight, which we will fix. Given that, do you have any further concerns about signing the CLA?

**Comment by robdimsdale** [comment]
The LICENSE file has been added.

**Comment by asiragusa** [comment]
Ok thanks, it's more clear now.

I'll do the paperwork tomorrow :)

**Comment by karlkfi** [comment]
With the added flags, the naming scheme should probably be revisited.
How about:
- host/port (to listen on)
- dbHost/dbPort (to connect to)
- dbUser/dbPassword (to connect with)

File: server.go [hunk 1]
```diff
@@ -78,7 +96,7 @@ func main() {
 		panic(err)
 	}
 
-	db, _ := sql.Open("mysql", fmt.Sprintf("%s:%s@/", *mysqlUser, *mysqlPassword))
+	db, _ := sql.Open("mysql", fmt.Sprintf("%s:%s@tcp(%s:%d)/", *mysqlUser, *mysqlPassword, *mysqlHost, *mysqlPort))
```
Comment (maximilien) [inline server.go]: Hmm, why are we ignoring the `error` returned in `sql.Open(...)` here? Just curious. Shouldn't you just check for it and `panic(err)` if it occurred?

My .02

Comment (robdimsdale) [inline server.go]: Good point. I think this is outside of the scope of this pull request however. Opened issue https://github.com/cloudfoundry-incubator/galera-healthcheck/issues/3

**Comment by karlkfi** [comment]
We went ahead and both renamed our flags and added ones analogous to yours:
https://github.com/cloudfoundry-incubator/galera-healthcheck/commit/bc307dd1f6cd60e98d9ed7b770f63cbec91f66d2
